scripts/play_imagination_world.py

```diff
@@ -0,0 +1,155 @@
+import cv2
+import hydra
+import keyboard
+import rootutils
+import torch
+from omegaconf import DictConfig
+from torch import Tensor
+
+from ami.interactions.environments.actuators.vrchat_osc_discrete_actuator import (
+    ACTION_CHOICES_PER_CATEGORY,
+)
+from ami.models.bool_mask_i_jepa import BoolMaskIJEPAEncoder
+from ami.models.components.discrete_policy_head import DiscretePolicyHead
+from ami.models.components.fully_connected_fixed_std_normal import (
+    DeterministicNormal,
+    FullyConnectedFixedStdNormal,
+)
+from ami.models.components.multi_embeddings import MultiEmbeddings
+from ami.models.components.sioconv import SioConv
+from ami.models.components.stacked_features import (
+    LerpStackedFeatures,
+    ToStackedFeatures,
+)
+from ami.models.forward_dynamics import ForwardDynamcisWithActionReward
+from ami.models.i_jepa_latent_visualization_decoder import (
+    IJEPALatentVisualizationDecoder,
+)
+
+
+class KeyboardActionHandler:
+    def __init__(self) -> None:
+        self.initial_actions = {"MoveVertical": 0, "MoveHorizontal": 0, "LookHorizontal": 0, "Jump": 0, "Run": 0}
+        self.actions = self.initial_actions.copy()
+
+        self.key_map = {
+            "w": ("MoveVertical", 1),
+            "s": ("MoveVertical", 2),
+            "d": ("MoveHorizontal", 1),
+            "a": ("MoveHorizontal", 2),
+            ".": ("LookHorizontal", 1),
+            ",": ("LookHorizontal", 2),
+            "space": ("Jump", 1),
+            "shift": ("Run", 1),
+        }
+
+    def update(self) -> None:
+        self.actions = self.initial_actions.copy()
+
+        for key, (action, value) in self.key_map.items():
+            if keyboard.is_pressed(key):
+                self.actions[action] = value
+
+    def get_action(self) -> Tensor:
+        return torch.tensor(
+            [
+                self.actions["MoveVertical"],
+                self.actions["MoveHorizontal"],
+                self.actions["LookHorizontal"],
+                self.actions["Jump"],
+                self.actions["Run"],
+            ]
+        )
+
+
+@hydra.main(
+    config_path="../data/2024-09-17_04-47-37,195367.ckpt", config_name="launch-configuration.yaml", version_base="1.3"
+)
+def main(cfg: DictConfig) -> None:
```

Comment:
決め打ちでハイパラ書いていくのでhydraいらないと思います．
```suggestion
def main() -> None:
```

Reply:
対応しました